content/instructor/quick-resources/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-04-13
+title: 'Quick Resources Page'
+description: "Getting ready to record? Find the information you need on one page!"
+categories: ['instructor']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_screencasting.png'
+redirect_from:
+
+---
+This **Quick Resources Page** compiles technical specifications that you might need to reference as you begin recording your lessons!
+
+⚠️ First, check out this comprehensive course by John to get started: [Record Badass Screencasts for egghead.io](https://egghead.io/courses/record-badass-screencasts-for-egghead-io)
+
+|     |     |      |
+|:-----:|:-----:|:------:|
+|[Desktop Environment](#desktop-environment)|[Audio Environment](#audio-environment)|[Recording](#recording)|
+|[Editing Videos](#editing-videos)|[Exporting Videos](#exporting-videos)|
+
+### Desktop Environment
+- Use large fonts and high-contrast color schemes
+- Record fullscreen at **16:9 aspect ratio**
+- **HiDPI** mode on monitor: **1280x720 pixel** resolution
+    1. [How on Windows](screencasting/screen-setup#dpi-scaling-on-windows)
+    2. [How on MacOS](screencasting/screen-setup#access-hidpi-mode-on-macos)
+- Use **2:1 ratio** of **code:example**:
+![](https://lh5.googleusercontent.com/yFu1UkYEzFM_zbwuf6kxZU5WuUwc_zhkaJY6uDSwk32Tega5Mj6rNukMByPiNZseZVyQgxPcRVsB07yh5R6IkiBUHnkm4mtAB9XOGJ4wt3qX0ZYcKw1kTVp0VJ-qNb9wJyctddH4)
+  1. Want to reformat your code to fit the screen? Check out “[Reformat Code to Fit the Screen](https://egghead.io/lessons/egghead-reformat-code-to-fit-the-screen)”
+- [Prepare your desktop environment for screencasting code](https://howtoegghead.com/instructor/screencasting/screen-setup/)
+### Audio Environment
+- The [egghead pro audio equipment](screencasting/audio-equipment/)
+- Set up your egghead audio equipment:
+<ResponsiveEmbed src="https://egghead.io/lessons/tools-setup-egghead-audio-gear-to-record-lessons/embed" />
+### Recording
+- How to prepare your screen and microphone:
+<ResponsiveEmbed src="https://egghead.io/lessons/tools-prepare-to-record-screen-resolution-and-mic-check/embed" />
+- Recommended applications:
+  1. **Windows:** Camtasia
+  2. **Mac:** ScreenFlow or Camtasia
+- Record one thought at a time!
+<ResponsiveEmbed src="https://egghead.io/lessons/tools-record-one-thought-at-a-time/embed" />
```

Comment:
I'd generally favor a summary of the video with a link to the source (or a collapsable section that hides the player) because the cheatsheet is generally useful while the videos aren't really something you'd watch repeatedly.